ports/mimxrt/main.c

```diff
@@ -41,6 +42,10 @@ void board_init(void);
 int main(void) {
     board_init();
     tusb_init();
+    led_init();
+
+    // Enable board LED after initialization
+    led_state(MACHINE_BOARD_LED, 1U);
```

Comment:
I think it best not to turn on LEDs by default but rather leave them off.  That's how the stm32 boards work.